content/de/bare-metal-machine.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Bare Metal Machine
+status: Feedback erwünscht
+category: Technologie
+---
+
+## Was es ist
+
+Bare Metal beschreibt einen physischen Computer, speziell einen Server, welcher nur ein einziges Betriebssystem hat. 
+Diese Unterscheidung ist wichtig, da viele, wenn nicht sogar die meisten, Server virtuelle Maschinen sind.
+Ein dedizierter physischer Server ist in der Regel ein großer Computer mit eingebauter leistungsstarker Hardware. 
+Die Installation eines Betriebssystems und die Ausführung von Anwendungen auf dieser physischen Hardware ohne Einsatz einer Virtualisierung, wird als "Bare Metal" bezeichnet.
+
+## Welches Problem es löst
+
+Die Verknüpfung eines Betriebssystems mit einem physischen Computer ist das ursprüngliche Muster der Datenverarbeitung. 
+Die kompletten Ressourcen des physischen Computers stehen dem Betriebssystem direkt zur Verfügung. Aufgrund einer fehlenden Virtualisierung gibt es keine künstliche Verzögerung bei der Übersetzung von Betriebssystemanweisungen in Maschinencode.
```

Comment:
und der erste Satz hier wiederum eher ins wie?

Für diesen Abschnitt würde ich sowas wie das folgende vorschlagen:
"Bei Virtualisierungen kommt es zu künstlichen Verzögerung bei der Übersetzung von Betriebssystemanweisungen in Maschinencode. Falls die Verarbeitung eines Workloads einen schnellen Zugriff auf Hardwareressourcen erfordert, kann Bare Metal die richtige Lösung sein."